graphql-dgs-codegen-core/src/main/kotlin/com/netflix/graphql/dgs/codegen/generators/java/DataTypeGenerator.kt

```diff
@@ -362,6 +367,68 @@ abstract class BaseDataTypeGenerator(
         addFieldWithGetterAndSetter(fieldDefinition.type, fieldDefinition, javaType)
     }
 
+    private fun addBitsetField(javaType: TypeSpec.Builder) {
+        val fieldsPresent = Field("fieldsPresent", com.squareup.javapoet.TypeName.get(java.util.BitSet::class.java))
+        val fieldBuilder = FieldSpec
+            .builder(fieldsPresent.type, ReservedKeywordSanitizer.sanitize(fieldsPresent.name))
+            .addModifiers(Modifier.PRIVATE, Modifier.FINAL, Modifier.TRANSIENT)
```

Comment:
Why is this transient?

Reply:
The BitSet variable is set to be transient so that serialization of data classes is unchanged.